src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -139,7 +139,7 @@ pub fn time_trace_profiler_finish(file_name: &str) {
 // to LLVM or the feature detection code will walk past the end of the feature
 // array, leading to crashes.
 
-const ARM_WHITELIST: &[(&str, Option<Symbol>)] = &[
+const ARM_ALLOWLIST: &[(&str, Option<Symbol>)] = &[
```

Comment:
Nit: It looks like all of these are features, perhaps a better name would be `ARM_FEATURES` or `ARM_TARGET_FEATURES`? 

That would be in keeping with the function below `all_known_features()`.

Reply:
This was bugging me as well -- I didn't like the use of "allowlist" here (I also don't like the use of "whitelist"). I'd prefer `TARGET_FEATURES` and I was thinking that the list of features would be like `target_features_supported` or something like that. That is, this is what we're really looking for, right? What set of features are supported by the current target.

Reply:
(In particular, reading the diff, I found myself confused about the meaning of these lists -- like, what are they listing, and why would something be added to it? I had to read the code a bit to understand what was going on, but if it had had a name like `ARM_TARGET_FEATURES` or even `ARM_SUPPORTED_TARGET_FEATURES` I would have known right away.